examples/diffuser/run_diffuser.py

```diff
@@ -0,0 +1,155 @@
+import numpy as np
```

Comment:
probably the core of a pipeline or example.

Reply:
do you want a pipeline to be part of this PR? Or something we add in a small follow up PR?